src/main/scala/chisel3/util/circt/PlusArgsTest.scala

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package chisel3.util.circt
+
+import chisel3._
+import chisel3.experimental.{annotate, ChiselAnnotation, ExtModule}
+import chisel3.internal.Builder
+
+import circt.Intrinsic
+
+/** Create a module with a parameterized type which calls the verilog function
+  * $test$plusargs to test for the existance of the string str in the
+  * simulator command line.
+  */
+private class PlusArgsTestIntrinsic[T <: Data](gen: T, str: String) extends ExtModule(Map("FORMAT" -> str)) {
+  val found = IO(Output(UInt(1.W)))
```

Comment:
```suggestion
  val found = IO(Output(Bool))
```
? is there a reason to have this as 1-bit uint vs just a Bool?

Reply:
Is there a particular reason to?  I've heard rumors of Bool v.s. UInt(1.W) fights, but I haven't payed them much heed.